force_gromacs/io/file_tree_builder.py

```diff
@@ -1,14 +1,14 @@
 import logging
 import os
 
-from traits.api import List, Unicode
+from traits.api import List, Str
 
-from force_gromacs.core.base_gromacs_process import BaseGromacsProcess
+from force_gromacs.core.base_process import BaseProcess
 
 log = logging.getLogger(__name__)
 
 
-class GromacsFileTreeBuilder(BaseGromacsProcess):
+class FileTreeBuilder(BaseProcess):
```

Comment:
This is great. I was wondering if this builder should be a part of the BDSS core?

Reply:
I think so, without the `subprocess` dependency it's definitely generic enough to be used elsewhere

Reply:
https://github.com/force-h2020/force-bdss/pull/282